File: 351001/Rudzenia/RW.Domain/Entities/Author.cs
namespace RW.Domain.Entities;

public class Author
{
    public long Id { get; set; }
    public string Login { get; set; } = string.Empty;
    public string Password { get; set; } = string.Empty;
    public string FirstName { get; set; } = string.Empty;
    public string LastName { get; set; } = string.Empty;
    public Role Role { get; set; } = Role.CUSTOMER;
}

File: 351003/Kolbeko/app/models/author.py
from sqlalchemy import Column, BigInteger, String
from app.core.database import Base

class Author(Base):
    __tablename__ = "tbl_author"
    __table_args__ = {"schema": "distcomp"}

    id = Column(BigInteger, primary_key=True, index=True)
    login = Column(String(64), nullable=False)
    password = Column(String(128), nullable=False)
    firstname = Column(String(64), nullable=False)
    lastname = Column(String(64), nullable=False)
    
    role = Column(String(32), default="CUSTOMER", nullable=True)